.github/CODEOWNERS

```diff
@@ -9,4 +9,4 @@ po/it.po @detro @nicomazz @bznein
 po/ja.po @keiichiw @chikoski
 po/ko.po @jiyongp @jooyunghan
 po/pt-BR.po @rastringer @hugojacob
-po/zh-Hans.po @suetfei
+po/zh-Hans.po @suetfei @wnghl
```

Comment:
Great, thanks for volunteering! I'll send you an invite for the repository.

If you move this change to another PR, then I can approve it and assign you to review the other two Chinese PRs.

Reply:
Thanks! I‘ll do this right now